cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/OperatorsTest.scala

```diff
@@ -244,9 +252,94 @@ class OperatorsTest extends DocumentingTest {
           resultTable()
         }
       }
-      p("""Further information and examples regarding the use of regular expressions in filtering can be found in <<query-where-regex>>.
+      p(
+        """Further information and examples regarding the use of regular expressions in filtering can be found in <<query-where-regex>>.
           |In addition, refer to <<query-operator-comparison-string-specific>> for details on string-specific comparison operators.""")
     }
+    section("Temporal operators", "query-operators-temporal") {
+      p(
+        """Temporal operators comprise:
+          |
+          |* adding a <<cypher-temporal-durations, _Duration_>> to either a <<cypher-temporal-instants, temporal instant>> or another _Duration_: `+`
+          |* subtracting a _Duration_ from either a temporal instant or another _Duration_: `-`
+          |* multiplying a _Duration_ with a number: `*`
+          |* dividing a _Duration_ by a number: `/`
+          |
+          |The following table shows -- for each combination of operation and operand type -- the type of the value returned from the application of each temporal operator:
+          |
+          |[options="header"]
+          ||===
+          || Operator | Left-hand operand | Right-hand operand | Type of result
+          || <<syntax-add-subtract-duration-to-temporal-instant, `+`>>      | _Duration_/Temporal instant  | Temporal instant/_Duration_ | The type of the temporal instant
```

Comment:
I think this should be split in two lines instead of having the / for lhs and rhs. I think this would make it clearer.